cves/CVE-2017-12611.yml

```diff
@@ -45,7 +45,9 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
```

Comment:
This reads very similarly to an NVD description. Please explain what terms like "RCE" and "Freemarker tags" mean.

Reply:
Added some additional explanation for freemarker tags and RCE.